src/background.js

```diff
@@ -1078,38 +1088,6 @@ function onMessageHandler(request, sender, callback) {
 	return false;
 }
 
-/**
- * Determine Antitracking configuration parameters based
- * on the results returned from the abtest endpoint.
- * @memberOf Background
- *
- * @return {Object} 	Antitracking configuration parameters
- */
-function getAntitrackingTestConfig() {
```

Comment:
@sammacbeth Can you review this section to make sure we've got the anti-track setting correct?